docs/channels-v1-doc.md

```diff
@@ -0,0 +1,79 @@
+- For the verifyToken config, it would be nice to have a button in the CDM ui to generate that random string for us.
+- All the doc about greeting text, get started and perisitent menu can be removed
+
+## Smooch
+
+Smooch is now at API 2. The UI didn't change much but it's now required to create the webhook yourself
+
+- In the same page where we generate the API key, there is an App ID section. The user needs to copy this now as well and write it as the "appId" config
+- The "secret" config is now named "keySecret"
+- Tell the user to copy this info into the textboxes in the UI, json config is not relevant to the cloud
+- Remove the step about restarting botpress
+- The user now needs to configure the webhook. They should go to the "Integrations" tab and scroll down to "API & Webooks" and click on the "Webhooks" box. There they create a webhook and paste the url given in the cdm UI. The version of the webhook has to be set to v2. They need to tick "Conversation message" and "Postbacks" in the "Basic Triggers" section when creating the webhook.
+- When they are finished creating the webhook, the need to copy the webhook secret displayed in the table into the "webhookSecret" config.
+
+## Telegram
+
+Telegram doesn't have any changes in configuration from the previous version.
+
+- We don't need to tell the user to setup an https endpoint since that's already done in the cloud
+- Tell the user to paste the token they generated with BotFather in the `botToken` textbox in the cdm UI instead of a config file
+- The webhook is still generated automatically. The UI says what the webhook endpoint is so we don't need to say it in the doc.
+
+## Slack
+
+Slack has changes in the config and the webhook routes were merged into a single route. I found that some steps were out of order, but the UI is still the same
+
+- There's no reason why setting the webhook in the "Interactivity & Shortcuts" section would be the first step we tell the user to do. They only reason this works is because setting this webhook does not check if the url is valid, but the other one does! Let's keep webhook configuration as the last step for all channels.
+- Need to do the Install App step before the "Allow users to send Slash commands" step. Also it's Basic Information -> "Install your app" not "Install app"
+- Change the steps to copy paste the info into textboxes on the cdm UI instead of a config file
+- `useRTM` setting doesn't exist anymore so we can remove the part that mentions it
+- NOW we tell the user to configure the webhooks in the "Interactivity & Shortcuts" and "Event Subscriptions". Let's tell the user to configure the "Event Subscriptions" webhook first, because this is where any error in configuration would be detected. Also we can tell the user to copy paste the webhook url provided in the ui. Both webhooks now have the same route, so we just say "copy the webhook url provided in the ui to both sections"
```

Comment:
```suggestion
- NOW we tell the user to configure the webhooks in the "Interactivity & Shortcuts" and "Event Subscriptions". Let's tell the user to configure the "Event Subscriptions" webhook first, because this is where any error in configuration would be detected. Also, we can tell the user to copy-paste the webhook URL provided in the UI. Both webhooks now have the same route, so we just say "copy the webhook URL provided in the UI to both sections"
```